src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -6,6 +6,8 @@
 import com.minecolonies.colony.buildings.BuildingTownHall;
 import com.minecolonies.colony.materials.MaterialSystem;
 import com.minecolonies.colony.permissions.Permissions;
+import com.minecolonies.colony.workorders.AbstractWorkOrder;
+import com.minecolonies.colony.workorders.WorkOrderBuild;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.workorders.WorkOrderBuild'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)